components/ui/vector/icons.remixicon

```diff
@@ -1 +1 @@
-search-line,contrast-2-fill,arrow-down-s-line,arrow-up-s-line,gas-station-fill,arrow-go-forward-line,arrow-left-s-line,arrow-right-s-line,checkbox-circle-line,command-line,contrast-2-line,error-warning-line,git-branch-line,github-fill,home-2-line,information-line,links-line,play-circle-line,question-line,reply-fill
+search-line,contrast-2-fill,arrow-down-s-line,arrow-up-s-line,gas-station-fill,arrow-go-forward-line,arrow-left-s-line,arrow-right-s-line,checkbox-circle-line,command-line,contrast-2-line,error-warning-line,git-branch-line,github-fill,home-2-line,information-line,links-line,play-circle-line,question-line,reply-fill,close-large-line,add-fill
```

Comment:
Also don't know what this does

Reply:
Same thing as [here](https://github.com/smlxl/evm.codes/pull/322#discussion_r1570396545)